src/ontology/uberon-edit.obo

```diff
@@ -101766,6 +101767,7 @@ property_value: external_ontology_notes "BTO may actually be a more generic stru
 id: UBERON:0005402
 name: philtrum
 def: "The vertical groove found on the median line of the upper lip, extending to the nose." [MGI:anna, MP:0012256, Wikipedia:Philtrum]
+comment: Philtrum is modelled as the space in of the groove rather than the surface structure itself.
```

Comment:
"... the space in of the groove... "
Don't quite understand what this means.

I agree that philtrum is a 'surface groove'. I don't think philtrum aligns with the text definition of ['anatomical space'](http://purl.obolibrary.org/obo/UBERON_0000464).

Reply:
hmmm interesting one, I agree, I think we need to move surface groove as immaterial anatomical entity instead of space